File: Codeforces/C++/problemset/Hulk.cpp
// Problem Link: https://codeforces.com/problemset/problem/705/A

#include <iostream>
using namespace std;
int main()
{
    long long n;
    cin >> n;
    for(long long i = 0; i < n; i++)
    {
        if(i%2 == 0)
            cout << "I hate ";
        else
            cout << "I love ";
        if((n - i) != 1)
            cout << "that ";
    }
    cout << "it";
    return 0;
}

File: Codeforces/Kotlin/src/main/kotlin/Hulk.kt
package problems

/* Problem Link: https://codeforces.com/problemset/problem/705/A */

fun main() {
    val n = readln().toInt()

    for (i in 0..<n) {
        if (i == n - 1) {
            if (i % 2 == 0) print("I hate it")
            else print("I love it")
        } else {
            if (i % 2 == 0) print("I hate that ")
            else print("I love that ")
        }
    }
}